website/server/controllers/api-v3/pushNotifications.js

```diff
@@ -0,0 +1,108 @@
+import { authWithHeaders } from '../../middlewares/api-v3/auth';
+import _ from 'lodash';
+import {
+  BadRequest,
+  NotAuthorized,
+} from '../../libs/api-v3/errors';
+
+let api = {};
+
```

Comment:
I would scope these two routes as POST /user/push-devices and DELETE /user/push-devices/:regId